packages/plugins/plugin-meeting/src/components/Call/Call.tsx

```diff
@@ -45,14 +45,14 @@ CallAudio.displayName = 'CallAudio';
 
 type CallRoomProps = {};
 
-const CallRoom: FC<CallRoomProps> = () => {
+const CallMain: FC<CallRoomProps> = () => {
```

Comment:
why `Main`? is this not also used in the sidebar?

Reply:
Changed to Call.Grid